lib/tasks/yalphabetize.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+namespace :yalphabetize do
+  task :run do
```

Comment:
Rails/RakeEnvironment: Include `:environment` task as a dependency for all Rake tasks.